shared/src/utils/type-state-selector-map.ts

```diff
@@ -0,0 +1,15 @@
+export const typeSelectorMap = {
+    alarmGroup: 'alarmGroups',
+    client: 'clients',
+    hospital: 'hospitals',
+    mapImage: 'mapImages',
+    material: 'materials',
+    patient: 'patients',
+    personnel: 'personnel',
```

Comment:
FYI: I regret this naming decision. Initially we had `personnel` and `personnels`, but changed it, after realising that `personnels` isn't a valid plural in the English language.
In hindsight, I believe this should have been a conscious neologism to still have the destination in variable names.

```ts
const personnel: Personnel;
const personnels: Personnel[];
```

Or using a synonym to personnel...

Reply:
Since we have to use the mapping between singular and plural anyway, I don't think that `personnel` as plural is a big problem.

If we *would* use `personnels`, we could just append an `s`, but this would break as soon as we've got something like `category`/`categories`.